.github/workflows/ci.yml

```diff
@@ -23,9 +22,9 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     - name: Install Python dependencies
-      run: python -m pip install --prefer-binary cython~=3.0 ${{ matrix.numpy-version }} scipy matplotlib "pyopencl[pocl]>=2022.2.4"
+      run: python -m pip install --prefer-binary setuptools cython~=3.1 numpy>=2 scipy matplotlib "pyopencl[pocl]>=2022.2.4"
     - name: Install Raysect from pypi
-      run: pip install raysect==0.8.1.*
+      run: pip install raysect==0.9.*
```

Comment:
I don't think there's a need any longer to install raysect separate from the rest of the dependencies. A long time ago it had to come from Github sometimes and PyPI other times, so made sense to deal with it differently, but it's about time this was cleaned up. Put it in the "Install Python dependencies" step.

Also, should be `raysect==0.9.1` since that's what we explicitly depend on in pyproject.toml and setup.py.

Reply:
I agree with you.
Moreover, we might be better off utilizing the dependency info that is configured in only one place (such as `setup.py` in this case).
Should we use the following command?:
```bash
python -m pip install -e ".[opencl]"
```